scripts/percona-release.sh

```diff
@@ -107,6 +110,22 @@ function check_specified_repo {
   fi
 }
 #
+function check_repo_availability {
+  if [[ "$2" == "-y" ]]; then
+    REPO_NAME=${3}
+  else
+    REPO_NAME=${2}
+  fi
+  reply=$(curl -Is http://repo.percona.com/${REPO_NAME}/ | head -n 1 | awk '{print $2}')
+  if [[ ${reply} == 200 ]]; then
+    if [[ ${REPOSITORIES} != "*${REPONAME}*" ]]; then
```

Comment:
vagrant@debian9:/etc/apt$ sudo percona-release setup pdpxc-8.0.20
404
* Disabling all Percona Repositories
ERROR: Unknown alias specification: pdpxc8.0.20
Available product aliases are: ps56 ps57 ps80 psmdb34 psmdb36 psmdb40 psmdb42 pxb80 pxc56 pxc57 pxc80 ppg11 ppg11.5 ppg11.6 ppg11.7 ppg11.8 ppg12 ppg12.2 ppg12.3 pdmdb4.2 pdmdb4.2.6 pdmdb4.2.7 pdmdb4.2.8 pdps8.0.19 pdpxc8.0.19 pdps8.0 pdpxc8.0

It seems that we still rely on the list of hardcoded aliases after checking the repo